src/main/java/com/spotify/reaper/service/RepairRunner.java

```diff
@@ -33,6 +33,7 @@
 import com.google.common.util.concurrent.Futures;
 import com.google.common.util.concurrent.ListenableFuture;
 import com.spotify.reaper.AppContext;
+import com.spotify.reaper.ReaperApplication;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.spotify.reaper.ReaperApplication'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)